app/views/prior_authority/steps/primary_quote/edit.html.erb

```diff
@@ -14,7 +14,7 @@
       <% if @form_object.draft? %>
         <%= suggestion_select form, :service_type_autocomplete, PriorAuthority::QuoteServices.values, :to_sym, :translated, width: 20,
```

Comment:
We already call PriorAuthority::QuoteServices.values in the controller - do we need to call that twice?

Reply:
Probably not, but it's a static call so I didn't think it was worth changing